sdk/tools/log2lines/log2lines.c

```diff
@@ -345,6 +345,9 @@ translate_line(FILE *outFile, char *Line, char *path, char *LineOut)
     sep = strchr(s, ':');
     if (sep)
     {
+        // Full initialization, wrt following scanf("%x"). Workaround for i.e. Linux 64-bit.
```

Comment:
Let's get rid of the hack instead of working around it. Just make offset an `unsigned int` and remove all the casts. That's equivalent to the current code but not as hacky... if someone needs full 64 bit support later they can worry about fixing that.

Reply:
Done.